app/views/profiles/show.html.erb

```diff
@@ -1,20 +1,23 @@
 <div class="section">
   <div class="container">
-    <div class="media is-flex-wrap-wrap mb-3" style="row-gap:1em">
+    <div class="media mb-3" style="row-gap:1em">
       <figure class="media-left image">
         <img src="<%= profile_picture(@profile.email) %>" alt="Profile pic">
       </figure>
       <div class="media-content">
         <p class="has-text-weight-bold is-size-4"><%= @profile.name %></p>
         <p class="has-text-weight-semi-bold is-size-6"><%= "@#{@profile.handle}" %></p>
       </div>
-      <div class="media-right">
-        <%= friendship_button(@profile) %>
-      </div>
     </div>
+    <p><%= policy(@profile).show_details? ? @profile.bio : unauthorized_message(@profile) %></p>
     <div class="block">
       <p>
-        <%= policy(@profile).show_details? ? @profile.bio : unauthorized_message(@profile) %>
+        <%= link_to_if @your_friendship, @your_friendship %>
```

Comment:
This returns no link if `@your_friendship` is nil. I can pass @your_friendship with no name, because the Firendship model has a `to_s` method that generates the link text for me.